packages/react/src/components/Authenticator/hooks/useAuthenticator/index.tsx

```diff
@@ -89,6 +89,16 @@ export type Selector = (context: AuthenticatorContext) => Array<any>;
 
 export const useAuthenticator = (selector?: Selector) => {
   const { service } = React.useContext(AuthenticatorContext);
+
+  if (!service) {
+    console.error(
+      'No `Authenticator.Provider` was found above where `useAuthenticator` hook is. ' +
+        'Please ensure you wrap your App with `Authenticator.Provider` like so: ' +
+        'https://ui.docs.amplify.aws/components/authenticator#useauthenticator-hook'
+    );
+    return null;
```

Comment:
Wasn't sure whether we should `throw` or just return `null` here. 

I went with `null` to avoid breaking customer apps unexpectedly. But on the other hand, it also makes sense to `throw` because not having `Provider` will break their app anyway. Up to discussion!

Reply:
👍